packages/next/build/babel/plugins/amp-attributes.ts

```diff
@@ -1,6 +1,6 @@
 import { NodePath, PluginObj, types } from '@babel/core'
 
-export default function AmpAttributePatcher(...args: any): PluginObj {
```

Comment:
Not sure if this is being extended somehow on some other place, but otherwise `args` wasn't used here.